frontend/pages/ManageControlsPage/MacOSSettings/cards/CustomSettings/components/DeleteProfileModal/DeleteProfileModal.tsx

```diff
@@ -5,11 +5,16 @@ import { AppContext } from "context/app";
 import Modal from "components/Modal";
 import Button from "components/buttons/Button";
 
+interface IDeleteProfileProps {
+  profileId: number;
+  profileName: string;
+}
+
 interface DeleteProfileModalProps {
   profileName: string;
   profileId: number;
   onCancel: () => void;
-  onDelete: (profileId: number) => void;
+  onDelete: ({ profileId, profileName }: IDeleteProfileProps) => void;
```

Comment:
profileName needed to trigger updating the config if we delete "Disk encryption"